kinetic/distribution.yaml

```diff
@@ -9642,6 +9642,11 @@ repositories:
       url: https://github.com/ros-geographic-info/open_street_map.git
       version: master
     status: maintained
+  open_vins:
+    doc:
+      type: git
+      url: https://github.com/rpng/open_vins
```

Comment:
The top level license file only mentions GPLv3, though one package.xml indicates MIT license ([ov_data](https://github.com/rpng/open_vins/blob/83ffb88ad35586d86bddc1e041094f4cd3d400df/ov_data/package.xml#L15)).

Could you make the information consistent? (i.e. make the top level license file mention all used licenses or update ov_data package.xml if that was a mistake).

Reply:
@goldbattle I'm assuming those the licensing for those datasets is GPLv3, correct?

Reply:
These are files that we have generated, the datasets each have their own licenses.

Reply:
@goldbattle I think this license question is still open. The top level license file should ideally describe each license in the repository and where it applies.

Reply:
This has been updated on our master:
https://github.com/rpng/open_vins/blob/master/ov_data/package.xml